packages/consent/consent.1.1.1/descr

```diff
@@ -0,0 +1 @@
+Chemoinformatics software for Ligand-Based Virtual Screening (LBVS) using consensus queries (small data; no machine learning).
```

Comment:
IMHO: You may want to have a shorter summary line and (after a blank line) have a longer description with links for people—like me—who would need more information to understand what the package does.  (This is obviously no blocker for merging, just a small comment.)

Reply:
done